flang/test/Lower/intrinsic-procedures/ISHFTC.f90

```diff
@@ -0,0 +1,43 @@
+! RUN: bbc -emit-fir %s -o - | FileCheck %s
```

Comment:
Combine with previous one since that also includes an ishftc test?

Reply:
It's a duplicate. I will remote `ishftc` from ISHFT.f90